hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestDataUtil.java

```diff
@@ -23,9 +23,7 @@
 import java.util.HashMap;
 import java.util.Scanner;
 
-import org.apache.hadoop.hdds.client.ReplicationConfig;
-import org.apache.hadoop.hdds.client.ReplicationFactor;
-import org.apache.hadoop.hdds.client.ReplicationType;
+import org.apache.hadoop.hdds.client.*;
```

Comment:
This change is not needed.
You can consider updating IntelliJ settings to not make wildcard imports.